src/Server/TextDocument.php

```diff
@@ -265,7 +265,15 @@ public function references(
     public function definition(TextDocumentIdentifier $textDocument, Position $position): Promise
     {
         return coroutine(function () use ($textDocument, $position) {
-            $document = yield $this->documentLoader->getOrLoad($textDocument->uri);
+            $documentLoader = $this->documentLoader;//->getOrLoad($textDocument->uri);
+            $document = null;
+            if (isset($documentLoader->documents[$textDocument->uri])) {
```

Comment:
the difference is that
getOrLoad won't do
```
$documentLoader->documents[$textDocument->uri] = $document; 
```

so in the case that you query this API very frequently in one document, the performance would suffer greatly